packages/sui-lint/sass-lint.yml

```diff
@@ -1,112 +0,0 @@
-files:
```

Comment:
This is a breaking change for those repositories relying in this deprecated configuration.